core-services/release-controller/_releases/release-ocp-4.3-ci.json

```diff
@@ -24,7 +24,6 @@
           "libvirt-installer",
           "baremetal-machine-controllers",
           "baremetal-operator",
-          "baremetal-runtimecfg",
```

Comment:
So 4.3 code was fixed?

Reply:
It was never broken - https://github.com/openshift/baremetal-runtimecfg/blob/release-4.3/Dockerfile doesn't use OSP repos